tools/build/esbuild.js

```diff
@@ -49,15 +53,21 @@ async function build(packageJson) {
     // Fixes issues related to SSR (website builds)
     platform: 'node',
     bundle: true,
-    // Sets the target environment so the code is changed into a format that
-    // works  with node12 and the listed browsers
+    /**
+     * Sets the target environment so the code is changed into a format that
+     * works  with node12 and the listed browsers
+     */
     target: ['chrome66', 'firefox58', 'safari11', 'edge79', 'node12.19.0'],
     define: {
       'process.env.NODE_ENV': `"${process.env.NODE_ENV}"`,
     },
     external,
   };
 
+  const versionInjectorConfig = {
+    packageJsonPath: path.join(__dirname, '../../packages/paste-core/core-bundle/package.json'),
```

Comment:
question/suggestion: can we do this for all the individual packages too? In case people have core installed, but also use something like theme or button individually as well? Or is that a limitation right now?

Reply:
That's the dream. What I wanted initially was a DOM node that looked like:

```html
<div data-paste-component="@twilio-paste/button" data-paste-component-version="9.0.0" data-paste-core-version="19.0.1" />
```

but the only way I could think about doing that was going into each component and tagging the outer most dom node in each package... and that seemed like a lot of work...

I can't think of a way to do it centerally because Box source isn't bundled into each package, it doesn't know anything about button using it, for example